### ISSUE TITLE
Is it possible to add vtkAxesActor ?

### ISSUE THREAD

**Issue by ericbdaniels** [plotly/dash-vtk#37]
Question and/or feature request -

Is it possible to add an axes actor/ orientation widget ?
Something along these lines: https://kitware.github.io/vtk-js/examples/InteractiveOrientationWidget.html

It is unclear to me how to go about this with the available dash components. Any guidance appreciated.

Thank you

**Comment by jourdain** [comment]
It is not possible as of right now but could be added. 

Could you describe what you would like to have in term of freedom? This will dictate how much work could be required for such implementation. Especially since in that example we have an interactive widget.

Here is a list of possible way of exposing it.

```
<View orientationAxes={true} ... />
```

or 

```
<View>
  <Viewport layer="1" position={[0, 0]} size={[0.1, 0.1]} interactive={false}>
      <GeometryRepresentation>
         <PolyData ... />
      </GeometryRepresentation>
  </Viewport>
</View>
```

or ?

Thanks,

Seb

**Comment by ericbdaniels** [comment]
Forgive me, that may have been an overly complex example to use as it is the interactive version. 

I believe either of those solutions would be valuable, though my main interest is in the first example.
From a python perspective I'm imagining something simple:

`dash_vtk.View(id="vtk-view", orientation_axes=True, children=...`

The interactive functionality is not particularly important for my use case though may be valuable to others.

**Comment by jlevy44** [comment]
I would find this to be valuable

**Comment by jlevy44** [comment]
Whichever is simplest.

**Comment by jlevy44** [comment]
Although I also just manually added axes, which works for me

**Comment by jourdain** [comment]
Actually that feature is already available since react-vtk-js@1.3.0 while dash-vtk is at 1.5.1.

Just need to add to the view `showCubeAxes=True`.
